graphql/concrete_notification/fields.ts

```diff
@@ -99,3 +101,53 @@ export class ExtendedFieldsConcreteNotificationResolver {
         return Object.values(byCampaign);
     }
 }
+
+@ObjectType()
+class ConcreteNotificationFormatted {
```

Comment:
`extends ConcreteNotification`, if the type is only used as an "output" (as the result of a Resolver) then adding FieldResolvers for the respective types would be enough (i.e. `FieldResolver(returns => String) headline(@Root() concreteNotification: ConcreteNotification) { ... }`. `@ObjectType`s make sense for inputs (types of arguments), or when returning an entity which is always accessed as a whole (thus there is no need for dedicated field resolvers). For the concrete notification, this is not really the case as not everyone wants to "render" concrete notifications (especially the body, it might also make sense to just query the headline in some cases)